scripts/sysman/frmwareSecurityVersion.yml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for firmware security version"
+version: "1.9"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Anti roll-back security firmware version"
+version: "1.9"
+name: $S_ANTI_ROLLBACK_SECURITY_VERSION_EXP_NAME
+value: '"$XS_extension_anti_rollback_security_version"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Anti rollback security version Extension Version(s)"
+version: "1.9"
+name: $s_firmware_security_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get anti roll-back security firmware version number of the current running firmware"
```

Comment:
IIRC, the IGSC team mentioned the number here is the version number that would get commited when the user calls "SetSecurityVersion" which is the trigger to write the number right?

also, do we need another out param or another API to provide a "min allowed version number"

Reply:
GetSecurityVersionExp returns the 'min version allowed', which comes from the fuses